bitrixSM/internal/services/command/help.go

```diff
@@ -1,7 +1,10 @@
-package command
+package bot_command
 
-func Help(args []string) error {
+import "log/slog"
+
+func Help(log *slog.Logger) string {
 	const commandList = `/create-task - создает задание. список аргументов: 
```

Comment:
nice) but I don't understand why do you need this, maybe it will be easier to create a simple doc, or even use comments in you swagger API spec